constraint/pkg/regorewriter/regorewriter_test.go

```diff
@@ -366,14 +376,14 @@ is_foo(name) {
 			libSrcs: map[string]string{
 				"lib/my_lib.rego": RegoSrc("lib"),
```

Comment:
Replace uses of `RegoSrc` with the strings it returns. As-is it's difficult to understand what the tests are doing.

Reply:
Add this test case to `TestRegoRewriteLib`

Reply:
I'm confused. If `TestRegoRewriteLib` tests the same functionality, then we can delete the tests which use RegoSrc. In either case, we shouldn't have any uses of `RegoSrc` left.

Reply:
`TestRegoRewriteLib` and `TestRegoRewriteEntryPoint` separately cover AddLib/AddEntryPoint then Rewrite the source, used the same source content from `TestRegoRewriter`. We can probably either delete `TestRegoRewriter` or remove the template usage.

Reply:
Great! Then let's delete it now.